### DISCUSSION TITLE
Running reactomeGSA for species other than human

### DISCUSSION THREAD

**Discussion by Ankita-1211** [reactome/ReactomeGSA#37] [Q&A] [answered]
Hi
 
I am following this tutorial **https://bioconductor.org/packages/release/bioc/vignettes/ReactomeGSA/inst/doc/analysing-scRNAseq.html** to run reactomeGSA for my scRNA data.

I have a question if I am running reactomeGSA for **species other than humans** how am I supposed to specify the same?
How does reactomeGSA take care of this while running?

**Comment by jgriss** [comment, marked as answer]
Hi @Ankita-1211,

Thanks a lot for your interest in ReactomeGSA!

As you may know, Reactome is a manually curated pathway database. In the curation process, all pathways are only curated for Human. The Reactome builds for all other species are created by automatically mapping the respective human identifiers to their homologous.

In ReactomeGSA we do this automatically as part of our mapping step. Since Reactome internally only uses UniProt identifiers, we always have to map all other identifier systems to UniProt. During that step, we already perform that "species conversion".

This is also the reason why you will always only see "human" pathways in your ReactomeGSA results. The net result is the same since Reactome is only curated for human pathways. Therefore, when using Reactome (irrespective of the analysis function), you always have to assess whether you can expect that the respective pathway is comparable to its human counterpart in your organism of interest.

Does that make sense?

Kind regards,
Johannes
